src/gflags.h.in

```diff
@@ -429,11 +429,57 @@ extern GFLAGS_DLL_DECL void ShutDownCommandLineFlags();
 // people can't DECLARE_int32 something that they DEFINE_bool'd
 // elsewhere.
 
+enum FlagValueType {
```

Comment:
I had to move this enum to a public place in header file, because FlagRegisterer class shouldn't be a template class IMHO. Implementation of the FlagRegisterer ctor uses a lot of classes internal to `gflags.cc` module, but a template should be a header-only.